http/src/main/java/io/micronaut/http/cookie/CookieConfiguration.java

```diff
@@ -63,4 +63,9 @@
      * @return The max age to use for the cookie
      */
     Optional<TemporalAmount> getCookieMaxAge();
+
+    /**
+     * @return return the SameSite to use for the cookie.
+     */
+    Optional<SameSite> getCookieSameSite();
```

Comment:
Make a default method that returns `Optional.empty()` to avoid breaking API change

Reply:
Done